x-pack/legacy/plugins/infra/public/components/logging/log_text_stream/column_headers.tsx

```diff
@@ -83,13 +89,16 @@ const LogColumnHeadersWrapper = euiStyled.div.attrs({
   justify-content: flex-start;
   overflow: hidden;
   padding-right: ${ASSUMED_SCROLLBAR_WIDTH}px;
+  border-bottom: ${props => props.theme.eui.euiBorderThin};
+  box-shadow: 0 2px 2px -1px ${props => transparentize(0.3, props.theme.eui.euiColorLightShade)};
```

Comment:
I think we could use the `euiSlightShadow` mixin here.

Reply:
> I think we could use the `euiSlightShadow` mixin here.

I'm not sure that can be done with the CSS-in-JS library we use. I'll give it a shot